keras_cv/models/backbones/csp_darknet/csp_darknet_backbone.py

```diff
@@ -387,7 +387,7 @@ def presets(cls):
     ),
 )
 setattr(
-    CSPDarkNetXBackbone,
+    CSPDarkNetXLBackbone,
     "__doc__",
     ALIAS_DOCSTRING.format(
         name="X",
```

Comment:
This should become XL to keep the docstring correct

Reply:
Done.